src/main/java/org/radarcns/redcap/managementportal/Subject.java

```diff
@@ -124,16 +123,18 @@ public String getHumanReadableIdentifier() {
     }
 
     /**
-     * Gets the project attribute (e.g. tag) associated with the given {@link String} key.
+     * Gets the subject attribute (e.g. tag) associated with the given {@link String} key.
      * @param key {@link String} tag key
      * @return {@link String} value associated with the given key
      */
     @JsonIgnore
     public String getAttribute(String key) {
-        Optional<Tag> tag = attributes.stream()
-                .filter(item -> item.getKey().equals(key)).findFirst();
 
-        return tag.isPresent() ? tag.get().getValue() : null;
+        if (attributes.getHumanRedableIdentifier() != null) {
```

Comment:
With attributes changed to `Map<String, String>` this can of course just become `return attributes.get(key);`.